torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -298,10 +298,12 @@ def apply_tp(model, world_mesh, parallel_dims, job_config: JobConfig):
     """
 
     tp_mesh = world_mesh["tp"]
+    # Parallel styles for transformer block linear weights may be different for
```

Comment:
These are only used for transformer block weights, so we can rename this to differentiate from the `ColwiseParallel` etc. for the other parameters.